src/states/elder/mod.rs

```diff
@@ -1112,8 +1059,11 @@ impl Elder {
             return;
         }
 
-        let (difficulty, target_size) = if self.disable_resource_proof
-            || self.network_service.is_peer_hard_coded(new_pub_id)
+        // TODO (quic-p2p): quic-p2p doesn't currently have an API to check if a contact is
```

Comment:
Given the code I reviewed in quic-p2p, I'm assuming this TODO will be addressed in this PR (?)

Reply:
Yes, as soon as https://github.com/maidsafe/quic-p2p/pull/53 is merged I'll push a commit to address this.